### DISCUSSION TITLE
add contact to multiple devices

### DISCUSSION THREAD

**Discussion by ariel256** [netbox-community/netbox#12942] [Q&A]
Hi,
is there a way to add contact/s to many devices in a single operation? or it must be done manually device after device?

**Comment by pobradovic08** [comment]
The easiest way I could think of is to use `/api/tenancy/contact-assignments/` API endpoint to create assignments for each device. If you're not caomtable with writing scripts, you can use Postman with prepared list of contact and device IDs (in a .csv file for example) to avoid doing it manually one device at the time.

**Comment by stavr666** [comment]
There is a Feature Request, to allow bulk import and update from UI: #11307 

Still "under review", tho.

Right now only possible from API, like suggested earlier.
